rules/falco_rules.yaml

```diff
@@ -2381,13 +2381,15 @@
     WARNING
   tags: [process, mitre_persistence]
 
-- rule: Delete Bash History
-  desc: Detect bash history deletion
+- rule: Delete or rename shell history
+  desc: Detect shell history deletion
   condition: >
-    ((spawned_process and proc.name in (shred, rm, mv) and proc.args contains "bash_history") or
-     (open_write and fd.name contains "bash_history" and evt.arg.flags contains "O_TRUNC"))
+    modify and evt.arg[1] contains "bash_history" or
```

Comment:
the `modify` works fine for the `mv` or `rm` scenarios, however, there is another scenario is not covered: write with `O_TRUNC` flag.

Reply:
You mean open followed by a write with `O_TRUNC` right?